migration-guide/redhat-to-ibm.adoc

```diff
@@ -0,0 +1,426 @@
+The most common ones are highlighted below, but download what makes the most sense for your environment. 
+
+.{PRODUCT_SHORT} Product Archives
+image::bamoe-downloads.png[Product Archives]
+
+The following archives are tyipcally used in migrating to {PRODUCT_SHORT}:
+
+.Common Product Archives
+|===
+| Product Archive | Description
+
+| ILMT_for_IBAMOE_**{VERSION**}.zip   
+| This archive is used by IBM ILMT in order to measure the usage deployed in an environment of the product
+
+| bamoe-**{VERSION**}-add-ons.zip   
+| This archive contains things like the Database Migration Scripts (found at path /bamoe-**VERSION**-add-ons/process-migration/upgrade-scripts/**Database**)
+
+| bamoe-**{VERSION**}-business-central-eap7-deployable.zip   
+| This archive is required if you are running Business Central on your JBoss Enterprise Application Platform (EAP)
+
+| bamoe-**{VERSION**}-kie-server-ee8.zip   
+| This archive is required if you are running running KIE Server on JBoss EAP
+
+| bamoe-**{VERSION**}-maven-repository.zip   
+| This archive is required if you need to have a disconnected loading of the Maven repository in your environment
+
+| bamoe-**{VERSION**}-offliner.zip   
+| This archive is required in order to help build the Maven repository in an offline fashion
+
+| bamoe-**{VERSION**}-update.zip 
+| This archive includes all of the updates that get applied with the overall product set. 
+
+|===
+
+[NOTE]
+====
+In the table above, you should replace **VERSION** with the version of {PRODUCT_SHORT} you are migrating to, such as 8.0.3.
+====
+
+_Now that we have those files we can start to migrate our environment._
+
+=== Step 2: Update Maven Configuration
+
+{MAVEN_SITE_URL_BASE}[Apache Maven] is a software project management and comprehension tool.
+Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information.
+{PRODUCT_LONG} relies on Maven in order to build various types of deployable artifacts, such as decision service applications.
+For a quick start see the {MAVEN_SITE_URL_GETTING_STARTED}[Maven Quick Start Guide].
+
+=== Update Maven Settings
+
+The settings element in the *_settings.xml_* file contains data used to define values which configure Maven execution in various ways, like the *_pom.xml_*, but should not be bundled to any specific project, or distributed to an audience.
+These include values such as the local repository location, alternate remote repository servers, and authentication information.
+There are two locations where a settings.xml file may live:
+
+- **The Maven install**: *_${maven.home}/conf/settings.xml_*
+- **A user's install**: *_${user.home}/.m2/settings.xml_*
+
+[NOTE]
+====
+The former `settings.xml` are also called global settings, the latter `settings.xml` are referred to as user settings.
+If both files exists, their contents gets merged, with the user-specific `settings.xml` being dominant.
+====
```

Comment:
same comment I did in the previuos document that has this content somewhat duplicated